solid/src/http.tsx

```diff
@@ -146,34 +146,24 @@ async function queryActivityCubeFullTimeframe(): Promise<boolean> {
     return false;
   }
 
-  const url = `/api/v1/activity?why=cube&database_list=(${
-    state.database_list.join("|") //
-  })&start=${
-    `now-${state.timeframe_ms}ms` //
-  }&end=${
-    "now" //
-  }&step=${
-    state.interval_ms //
-  }ms&limitdim=${
-    state.activityCube.limit //
-  }&limitlegend=${
-    state.activityCube.limit //
-  }&legend=${
-    state.activityCube.uiLegend //
-  }&dim=${
-    state.activityCube.uiDimension1 //
-  }&filterdim=${
-    state.activityCube.uiFilter1 === DimensionName.none ||
-    !state.activityCube.uiFilter1Value
+  const url = `/api/v1/activity?why=cube&database_list=(${state.database_list.join("|") //
+    })&start=${`now-${state.timeframe_ms}ms` //
+    }&end=${"now" //
```

Comment:
These changes need to have `prettier` run on them because it will revert what yuo've written here. If that's hard for you, then Radcliffe or I can do it quickly. We should add `prettier` as a part of a commit hook.

Reply:
Applied `prettier`. It makes sense to add it as a commit hook.